gallery/src/components/NxCodeSnippet/NxCodeSnippetPage.tsx

```diff
@@ -0,0 +1,114 @@
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>content</NxTableCell>
+              <NxTableCell>string</NxTableCell>
+              <NxTableCell>Yes</NxTableCell>
+              <NxTableCell></NxTableCell>
+              <NxTableCell>
+                The text content to display within the text area and for which to enable copying to the clipboard
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>onCopyUsingBtn</NxTableCell>
+              <NxTableCell>Function () =&gt; void</NxTableCell>
+              <NxTableCell>No</NxTableCell>
+              <NxTableCell></NxTableCell>
+              <NxTableCell>
+                A callback which fires after the copy initiated by the Copy to Clipboard button completes.
+                Note that copies initiated in other ways (e.g. Ctrl-C) do not trigger this callback.
+                See also the{' '}
+                <a href="https://developer.mozilla.org/en-US/docs/Web/API/Element/copy_event"
+                   target="_blank"
+                   rel="noopener">
+                  native copy event
+                </a>
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>HTML <code className="nx-code">&lt;div&gt;</code> Attributes</NxTableCell>
+              <NxTableCell>
+                <a target="_blank"
+                   rel="noopener"
+                   href="https://developer.mozilla.org/en/docs/Web/HTML/Element/div">
+                  HTML div Attributes
+                </a>
+              </NxTableCell>
+              <NxTableCell>No</NxTableCell>
+              <NxTableCell>N/A</NxTableCell>
+              <NxTableCell>
+                <code className="nx-code">NxCodeSnippet</code> supports any HTML attribute that's normally
+                supported by <code className="nx-code">&lt;div&gt;</code>.
+              </NxTableCell>
+            </NxTableRow>
+          </NxTableBody>
+        </NxTable>
+      </GalleryDescriptionTile>
+
+      <GalleryExampleTile title="Simple Example"
+                          liveExample={NxCodeSnippetExample}
+                          codeExamples={nxCodeSnippetCode}>
+        An <code className="nx-code">NxCodeSnippet</code> with minimal props: just some content which can be
+        copied to the clipboard.
+      </GalleryExampleTile>
+
+      <GalleryExampleTile title="More Complex Example"
+                          id="nx-code-snippet-example"
+                          liveExample={NxCodeSnippetComplexExample}
+                          codeExamples={nxCodeSnippetComplexCode}>
+        An <code className="nx-code">NxCodeSnippet</code> demonstrating optional
+        props: <code className="nx-code">sublabel</code> and <code className="nx-code">onCopyUsingBtn</code>.
+      </GalleryExampleTile>
+    </>
+  );
+}
+
```

Comment:
We might consider adding a test that has two of these next to each other since I know that will happen in nxrm in more than one location.

Reply:
> We might consider adding a test that has two of these next to each other since I know that will happen in nxrm in more than one location.

8b33296b